pkg/cmd/subcommand/install.go

```diff
@@ -349,6 +355,7 @@ func (i *Install) mergeConditions() {
 
 	if i.OpenFunctionVersion == "v0.4.0" {
```

Comment:
Should have a better way to handle this instead of hardcoding v0.4.0?
WithIngress and WithCertManager is specific to v0.4.0? @tpiperatgod @wanjunlei

Reply:
has been changed as follows:

```go
	if i.openFunctionVersion.Major() == 0 && i.openFunctionVersion.Minor() == 3 {
		i.WithIngress = false
		i.WithCertManager = false
	}

	if i.openFunctionVersion.Major() == 0 && i.openFunctionVersion.Minor() == 4 {
		i.WithIngress = false
		i.WithCertManager = true
	}
```

`WithCertManager` works in v0.4.*
`WithIngress` works in v0.5.* (still in the latest version).